### PR TITLE
Removing hopscotch in CMakeLists.

### PR DESCRIPTION
**Pull request by jlconlin**
This was causing NJOY21 to fail to compile.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -195,18 +207,6 @@ set( lipservice_Clang_Darwin_link_time_optimization_flags "-flto" )
...
-set( lipservice_Clang_Windows_subproject_flags  )
-set( lipservice_Clang_Windows_base_project_flags  )
-set( lipservice_Clang_Windows_profile_generate_flags "-fprofile-generate='${CMAKE_BINARY_DIR}/profiling'" )
-set( lipservice_Clang_Windows_link_time_optimization_flags "-flto" )
-set( lipservice_Clang_Windows_profile_use_flags "-fprofile-use='${CMAKE_BINARY_DIR}/profiling'" )
-set( lipservice_Clang_Windows_nonportable_optimization_flags "-march=native" )
-set( lipservice_Clang_Windows_static_flags "-static" )
 get_directory_property( is_subproject PARENT_DIRECTORY )
 
 if( NOT TARGET hopscotch-map-adapter )
```
Comment (whaeck) [inline CMakeLists.txt]: The error still occurs. You probably still had the directory in your subproject tree when you ran metaconfigure on the project.

Comment (jlconlin) [reply CMakeLists.txt]: Good catch. This is an indication of the fragility of our current process.

**Comment by jlconlin** [comment]
There is one or two inputs that take a `TAB1` record as input. We've used ENDFtk to facilitate this.

**Comment by jlconlin** [comment]
@whaeck I think I've removed the hopscotch dependency (really this time!) Sure wish we had AWS!

**Comment by whaeck** [comment]
I'm doing a clean clone/cmake now. As far as I can see, it looks to be functional now. CMake is still generating the make files. That appears to take quite a long time, especially compared to some of the other libraries (I don't know why).

**Pull request merged**